### DISCUSSION TITLE
What did I do wrong?

### DISCUSSION THREAD

**Discussion by EmaBixD** [illixion/vscode-vibrancy-continued#157] [Q&A] [answered]
Vibrancy does not work properly when switching between light/dark modes and when closing/restarting VSCode on Macbook.
Attached: my extension settings and what the panel looks like In these situations.

Settings: (and yes, "window.autoDetectColorScheme" is active)
![Screenshot 2024-08-18 alle 17 34 20](https://github.com/user-attachments/assets/6508a7eb-9cc5-4e2b-aabf-8d2327327839)

? Light mode:
![Screenshot 2024-08-18 alle 17 29 54](https://github.com/user-attachments/assets/b6537b29-fe1b-46e2-9527-c660034bc960)

? Dark mode:
![Screenshot 2024-08-18 alle 17 30 25](https://github.com/user-attachments/assets/a56d5f27-0894-4c00-94b4-0b23dd1bed14)

Has no one ever had this problem? I have the correct theme enabled but I can't get the extension to work properly at every startup despite following the instructions.

**Comment by illixion** [comment]
It doesn't appear that Vibrancy is active, please follow the installation instructions by pressing ⌘ + Shift + P and selecting **Reload Vibrancy**.

**Reply by illixion** [reply, marked as answer]
That is strange, there are no errors, only the success message asking to restart? If you could also provide your VSCode's version info from "About Visual Studio Code", that would be helpful. You can also try reinstalling VSCode, as that sometimes helps with issues like these and you won't lose anything.

**Reply by illixion** [reply]
Glad to hear that it works now, the issue you’ve mentioned is likely an oversight in the code, I’ll look into fixing it.
